spec/requests/api/v2/providers/courses/publish_spec.rb

```diff
@@ -135,6 +135,24 @@
                ))
         end
       end
+
+      context "with a new site_status" do
+        let(:site_status) { build(:site_status, :new) }
+
+        it "Successfully publishes the course" do
+          perform_enqueued_jobs do
+            expect(subject).to have_http_status(:success)
+          end
+
+          expect(course.reload.site_statuses.first).to be_status_running
+          expect(course.site_statuses.first).to be_published_on_ucas
+          expect(course.enrichments.first).to be_published
+          expect(course.enrichments.first.updated_by_user_id).to eq user.id
+          expect(course.enrichments.first.updated_at).to be_within(1.second).of Time.now.utc
```

Comment:
I generally prefer that we use Timecop to make the exact time knowable (since sometimes seconds varying is a sign something is broken), but I don't think it's a big deal here.

Reply:
:man_facepalming: i forgot Timecop, i'll add it before i merge :+1: